messages/de-DE.json

```diff
@@ -0,0 +1,384 @@
+    },
+    "categoryField": {
+      "label": "Kategorie"
+    },
+    "notesField": {
+      "label": "Notizen"
+    },
+    "selectNone": "Keine auswählen",
+    "selectAll": "Alle auswählen",
+    "shares": "Anteil(e)",
+    "advancedOptions": "Fortgeschrittene Aufteilungsoptionen…",
+    "SplitModeField": {
+      "label": "Aufteilungsart",
+      "evenly": "Gleich verteilt",
+      "byShares": "Ungleich – Nach Anteilen",
+      "byPercentage": "Ungleich – Prozentual",
+      "byAmount": "Ungleich – Nach Betrag",
+      "saveAsDefault": "Als Standardoptionen zur Aufteilung speichern"
+    },
+    "DeletePopup": {
+      "label": "Löschen",
+      "title": "Diese Ausgabe löschen?",
+      "description": "Willst du diese Ausgabe wirklich löschen? Diese Aktion kann nicht rückgängig gemacht werden.",
+      "yes": "Ja",
+      "cancel": "Abbrechen"
+    },
+    "attachDocuments": "Dokument hinzufügen",
+    "create": "Erstellen",
+    "creating": "Erstellt…",
+    "save": "Speichern",
+    "saving": "Speichert…",
+    "cancel": "Abbrechen"
+  },
+  "ExpenseDocumentsInput": {
+    "TooBigToast": {
+      "title": "Das File ist zu groß",
+      "description": "Die maximale File-Größe ist {maxSize}. Deine ist ${size}."
+    },
+    "ErrorToast": {
+      "title": "Fehler beim Hochladen des Dokuments",
+      "description": "Beim Hochladen des Dokuments ist etwas schiefgelaufen. Versuche es später nochmal oder wähle ein anderes File.",
+      "retry": "Wiederholen"
+    }
+  },
+  "CreateFromReceipt": {
+    "Dialog": {
+      "triggerTitle": "Ausgabe von Rechnungsbeleg erstellen",
+      "title": "Von Rechnungsbeleg erstellen",
+      "description": "Ausgabeninformationen von einem Foto einer Rechnung lesen.",
+      "body": "Lade ein Foto der Rechnung hoch und wir versuchen die Ausgabeinformationen zu extrahieren",
+      "selectImage": "Bild wählen…",
+      "titleLabel": "Titel:",
+      "categoryLabel": "Kategorie:",
+      "amountLabel": "Betrag:",
+      "dateLabel": "Datum:",
+      "editNext": "Als nächstes kannst du die Informationen zur Ausgabe editieren.",
+      "continue": "Weiter"
+    },
+    "unknown": "Unbekannt",
+    "TooBigToast": {
+      "title": "Das File ist zu groß",
+      "description": "Die maximale File-Größe ist {maxSize}. Deine ist ${size}."
+    },
+    "ErrorToast": {
+      "title": "Fehler beim Hochladen des Dokuments",
+      "description": "Beim Hochladen des Dokuments ist etwas schiefgelaufen. Versuche es später nochmal oder wähle ein anderes File.",
+      "retry": "Wiederholen"
+    }
+  },
+  "Balances": {
+    "title": "Bilanz",
+    "description": "Das sind die Beträge, die jedes Mitglied bezahlt oder empfangen hat.",
+    "Reimbursements": {
+      "title": "Vorgeschlagene Rückzahlungen",
+      "description": "Hier sind Vorschläge für optimierte Rückzahlungen zwischen Mitgliedern.",
+      "noImbursements": "Es sieht aus, als bräuchte deine Gruppe keine Rückzahlungen 😁",
```

Comment:
The group doesnt need the reimbursements at all. So "als seien in der Gruppe keine Rückzahlungen nötig" is a better way to phrase it in my view as it looks at the group members, not the group itself.

Reply:
I wanted to stay closer to the English translation but your version sounds better :)

Reply:
If German, we should do it properly. "Denglish" might not look professional. I fully agree with the comments and proposals of @tedzards509